cg/cli/workflow/balsamic/store.py

```diff
@@ -24,123 +29,83 @@ def store(context):
 
 
 @store.command()
-@click.argument("config-stream", type=click.File("r"), required=False)
+@click.argument("case_id")
+@click.option("--deliverables-file", "deliverables_file_path", required=False, help="Optional")
 @click.pass_context
-def analysis(context, config_stream):
+def analysis(context, case_id, deliverables_file_path):
     """Store a finished analysis in Housekeeper."""
+
     status = context.obj["db"]
-    tb_api = context.obj["tb_api"]
-    hk_api = context.obj["hk_api"]
+    case_obj = status.family(case_id)
 
-    if not config_stream:
-        LOG.error("provide a config, suggestions:")
-        for analysis_obj in tb_api.analyses(status="completed", deleted=False)[:25]:
-            click.echo(analysis_obj.config_path)
+    if not case_obj:
+        click.echo(click.style(f"Case {case_id} not found", fg="red"))
         context.abort()
 
-    new_analysis = _gather_files_and_bundle_in_housekeeper(
-        config_stream, context, hk_api, status, tb_api
-    )
-
-    status.add_commit(new_analysis)
-    click.echo(click.style("included files in Housekeeper", fg="green"))
-
+    if not deliverables_file_path:
+        root_dir = Path(context.obj["balsamic"]["root"])
+        deliverables_file_path = Path.joinpath(
+            root_dir, case_id, "analysis/delivery_report", case_id + ".hk"
+        )
+        if not os.path.isfile(deliverables_file_path):
```

Comment:
deliverables_file_path is an instance of Path it seems, do we need to use `os.path.isfile()` ? why not  `deliverables_file_path.is_file()` ?

Reply:
I got a warning that is_file is not defined on PurePath, worked around it instead of trying to understand what the problem was. What do you suggest?

Reply:
I've been annoyed by that warning as well! My suggestion here is that whatever works works 😄

Reply:
great